packages/tasks/package.json

```diff
@@ -48,9 +51,10 @@
 	],
 	"author": "Hugging Face",
 	"license": "MIT",
-	"devDependencies": {
-		"@types/node": "^20.11.5",
-		"quicktype-core": "https://github.com/huggingface/quicktype/raw/pack-18.0.17/packages/quicktype-core/quicktype-core-18.0.17.tgz",
-		"type-fest": "^3.13.1"
+	"tshy": {
+		"exports": {
+			"./package.json": "./package.json",
+			".": "./src/index.ts"
+		}
```

Comment:
that's the default, right?

Reply:
it set itself like that when running `pnpm build`, I didn't add it :)